server/renderer/index.js

```diff
@@ -7,8 +7,14 @@ function createNode(name) {
 }
 function createAttribute(key, value) {
   if (value === false) return '';
-
-  return `${key}="${value === true ? key : value}"`;
+  if (value === true) return `${key}="${key}"`;
+  const quoteStyle = /"/.test(value) ? "'" : '"';
+  let attr = `${key}=`;
+  attr += quoteStyle;
+  attr += value;
+  attr += quoteStyle;
+
+  return attr;
```

Comment:
```suggestion
  return `${key}="${value === true ? key : value.replace(/"/g, '&quot;')}"`;
```

Reply:
This seems to fix the problem. I can then `getAttribute` and `JSON.parse` the value

Reply:
Yeah, yeah, be thorough. ;) This looks good and is similar to how `ReactDOMServer.renderToString` handles attributes. I didn't see a case where we had `"` and `'` in a single JSON blob, so I didn't worry about it. 😂